Mathlib/Algebra/Category/AlgebraCat/Basic.lean

```diff
@@ -30,7 +30,6 @@ structure AlgebraCat where
   [isRing : Ring carrier]
   [isAlgebra : Algebra R carrier]
 
--- Porting note: typemax hack to fix universe complaints
```

Comment:
I don't know what this means

Reply:
We've to decided to keep these. This particular one isn't used anywhere, but the others in the family are used at various points to avoid universe unification failures. No need to keep a porting note.

Reply:
But I'll separately investigate whether we really need them still. Things have changed since we worked out this trick.

Reply:
All removed now in #22417 and #22418.